### PR TITLE
BRING-1: add a skeleton for bring project

### PR DESCRIPTION
**Pull request by kebann**
- added the license file
- generated the maven project with core deps
- added github workflows and default reviewers

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+Why is this change required? What problem does it solve?
+
+# How:
+
+Please describe how the change has been implemented
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes.
```
Comment (PavloChechehov) [inline .github/PULL_REQUEST_TEMPLATE.md:11]: spelling mistakes

File: README.md [hunk 2]
```diff
@@ -1,2 +1,26 @@
...
+    - [Context](#context)
+
+## Description
+
+### Features
+
+**Bring Framework** is an inversion of control and dependency injection framework. It allows you to declare what objects
+you need, and then it processes configuration, creates required objects, sets dependencies and brings objects that are
+ready
+to use.
```
Comment (PavloChechehov) [inline README.md:15]: redundant new line

**Review by PavloChechehov — APPROVED** [review]

**Review by amfibiotik — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (kebann) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: where?

File: README.md (hunk 2 above)
Comment (kebann) [reply README.md:15]: fixed

**Pull request merged**